### ISSUE TITLE
OSA Bug 1632040: RabbitMQ cluster killed during setup-hosts.yml

### ISSUE THREAD

**Issue by BjoernT** [rcbops/rpc-openstack#1482]
##### What type of issue is this? Please tick the appropriate box.
- [X] bug - existing functionality that does not work for you as documented
- [ ] feature - new functionality that you would like to see added

##### What is the expected behavior?

RabbitMQ cluster should not be dying for a extended time just because of a container restart
I assume the hostname change triggered an issue too.
The nova services for example did not reconnect at all, only once the nova playbook ran (over an hour into the upgrade)

##### What is actually happening?

See https://bugs.launchpad.net/openstack-ansible/+bug/1632040

##### What were you attempting to do when you encountered this error (give as much detail as possible)? 

Just running the upgrade script

##### How can we reproduce this issue?

Upgrade Liberty to Mitaka

##### How can we verify your issue is addressed?

No extended downtime (+20 minutes and +60 minutes) of rabbitmq and services

##### If you are using or know of a workaround please describe the necessary steps?

We ran rabbitmq-install why galera-install play was running to limit impact

##### What is your evaluation on the importance of this issue, and why?

Prio 2

----------------------------------------------------------------------------------------------------------------------------
###### For each pull request, update the branch line to reference it.
- [ ] master
- [ ] mitaka-13.1

##### Upgrade impact. Consider the impact on moving between major, minor and patch releases.
- [ ] This change does not impact upgrades
- [ ] This change addresses a potential upgrade impact
- [ ] The upgrade impact is being tracked by another issue.

The issue tracking the upgrade impact is:

##### Testing impact
- [ ] An issue/pull-request has been create or exists to track the testing gap, and is referenced below
- [ ] This change includes additional testing
- [ ] This change is tested by the existing setup

Additional testing links:

##### Release note
- [ ] This change includes a release note.
- [ ] This change does not require a release note.

##### Does your change require documentation?
- [ ] Yes, and I submitted an issue to [Docs issues](https://github.com/rackerlabs/docs-rpc/issues "Docs issues")
- [ ] Yes, and I contributed a documentation patch to [docs-rpc](https://github.com/rackerlabs/docs-rpc "docs-rpc")
- [ ] No documentation required for this change

Additional documentation links (please add links if you have ticked yes):
